src/resources/index.ts

```diff
@@ -1,9 +1,10 @@
 import {FrameworkConfiguration} from 'aurelia-framework';
+import {PLATFORM} from 'aurelia-pal';
 
 export function configure(config: FrameworkConfiguration) {
   config.globalResources([
-    './attributes/md-auto-focus',
-    './elements/nav-bar',
-    './elements/modals/add-todo-modal'
+    PLATFORM.moduleName('./attributes/md-auto-focus'),
+    PLATFORM.moduleName('./elements/nav-bar'),
+    PLATFORM.moduleName('./resources/elements/modals/add-todo-modal')
```

Comment:
my bad.... the resources folder can be removed in the path.

related to the PLATFORM change: I thought the problem of aurelia not finding add-todo-modal could be related to this. And according to https://github.com/jods4/aurelia-webpack-build/wiki/Managing-dependencies this needs to be done. However it did not help (but also did not harm and might have resulted in other problems later)